<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adminintration</class>
 <widget class="QDialog" name="Adminintration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBoxAdmin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Administration</string>
   </property>
   <widget class="QPushButton" name="pushButtonLogInfi">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>User Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RechnungInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Rechnung Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MitarbeitInfo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Mitarbeit Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RoleInfo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Role Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ProdukteInfo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Produkte Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LagerInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Lager Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoginERstellen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Login Erstellen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CreateMitarbeit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Mitarbeit Registriert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RechnungErstellen">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Rechnung Erstellen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRechnungErstellen">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Rechnung erstellen</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_DigitalClock">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>321</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Digital Clock</string>
   </property>
   <widget class="QLabel" name="label_DigitalClock">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clock</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DateTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_DatabaseConnecction">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Database</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_NameUser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Name User</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxSoftpace">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>661</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Softpace</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Qt/Logo/Logo.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Qt/Blaise-Passbilder/Blaise-Passbild_klein_sw.jpg</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
